src/api/routes/user.ts

```diff
@@ -17,11 +12,12 @@ import { ApiError, apiErrorCodes } from '../ApiError';
 import { DomainValidationError } from '../../domain/model/DomainValidationError';
 import { ResourceNotFoundError } from '../../domain/model/ResourceNotFoundError';
 import { UserNotFoundError } from '../../domain/model/user/UserRepository';
+import { UpdateUserCommand } from '../../presentation/dtos/users';
 
 export default () => {
   const route = new AsyncRouter();
 
-  route.get('/users/:id', isAuthenticated, async (req: AuthenticatedRequest, res: Response) => {
+  route.get('/:id', isAuthenticated, async (req: AuthenticatedRequest, res: Response) => {
```

Comment:
How does the rest of the path get established?  Or are we actually changing the path?